src/assets/css/material/_theme.scss

```diff
@@ -408,4 +408,18 @@
       border-bottom: 1px solid map.get($colors, tab-divider);
     }
   }
+
+  // Checkbox
+  .mat-checkbox-disabled {
+    &.mat-checkbox-checked,
+    &.mat-checkbox-indeterminate {
+      .mat-checkbox-background {
+        background-color: map.get($colors, disabled-checkbox);
+      }
+    }
+
+    .mat-checkbox-label {
+      color: map.get($colors, text);
```

Comment:
I think it would be helpful to change label text color so it also appears disabled.

Reply:
Should I reduce the transparency of the text?

Reply:
```suggestion
      color: map.get($colors, text-disabled);
```

Reply:
https://material.angular.io/components/checkbox/api

`mat-checkbox` already contains `disabled` property that should be used instead.